README.md

```diff
@@ -0,0 +1,73 @@
+# Jaeger Spark dependencies
+
+This is a Spark job that will collect spans from your datastore, analyze links between services,
+and store them for later presentation in the UI.
+
+This job parses all traces in the current day in UTC time. This means you should schedule it to run
+just prior to midnight UTC.
```

Comment:
not sure I follow this. The job we run internally is also based on UTC, but it runs _after_ UTC midnight and processes the _previous_ day.